### PR TITLE
DOCS-2244: Publishes Calico Enterprise 3.19.2

### PR DESCRIPTION
**Pull request by ctauchen**
Still needs:
* ~version updates~
* ~merge #1556 to this branch. @sridhartigera is going to:~
  * ~port changes to 3.19 docs~
  * ~fetch `publish/ce-3.19.2` change tracking branch to `publish/ce-3.19.2`, squash commits with git rebase -i~
  * ~confirm build passes (yarn build)~
  * ~change PR target branch to `publish/ce-3.19.2`~
  * ~tag @rene-dekker for review and merge~ @ctauchen reviewed
  * ~merge~
  * There won't be a deploy preview, so we'll just rely on the report of a clean build

### CODE REVIEW

File: calico-enterprise_versioned_docs/version-3.19-2/releases.json [hunk 1]
```diff
@@ -1,4 +1,255 @@
...
+      "eck-elasticsearch": {
+        "version": "7.17.21"
+      },
+      "elasticsearch": {
+        "image": "tigera/elasticsearch",
+        "version": "v3.19.2"
+      },
+      "elastic-tsee-installer": {
+        "image": "tigera/intrusion-detection-job-installer",
+        "version": "v3.19.2"
```
Comment (ctauchen) [inline calico-enterprise_versioned_docs/version-3.19-2/releases.json]: @danudey Should be changes here. CC @pasanw

File: calico-enterprise_versioned_docs/version-3.19-2/releases.json [hunk 2]
```diff
@@ -1,4 +1,255 @@
 [
+  {
+    "title": "v3.19.2",
+    "tigera-operator": {
+      "image": "tigera/operator",
+      "version": "v1.34.2",
```
Comment (ctauchen) [inline calico-enterprise_versioned_docs/version-3.19-2/releases.json]: @danudey Needs change here.

File: calico-enterprise_versioned_docs/version-3.19-2/releases.json [hunk 3]
```diff
@@ -1,4 +1,255 @@
...
+      },
+      "prometheus": {
+        "image": "tigera/prometheus",
+        "version": "v3.19.2"
+      },
+      "coreos-prometheus": {
+        "version": "v2.48.1"
+      },
+      "coreos-prometheus-operator": {
+        "version": "v0.70.0"
```
Comment (rene-dekker) [inline calico-enterprise_versioned_docs/version-3.19-2/releases.json]: Has been increased to v0.73.2

File: calico-enterprise_versioned_docs/version-3.19-2/releases.json [hunk 4]
```diff
@@ -1,4 +1,255 @@
...
+      },
+      "coreos-config-reloader": {
+        "version": "v0.70.0"
```
Comment (rene-dekker) [inline calico-enterprise_versioned_docs/version-3.19-2/releases.json]: Has been increased to v0.73.2

**Pull request merged**